cmd/portal/analytic_collect_count.go

```diff
@@ -54,11 +57,19 @@ var cmdAnalyticCollectCount = &cobra.Command{
 			DatabaseSchema: auditDBSchema,
 		}
 
+		dbPool := db.NewPool()
+		countCollector := analytic.NewCountCollector(context.Background(), dbPool, dbCredentials, auditDBCredentials)
+
 		interval := args[0]
 		switch interval {
 		case analytic.CollectIntervalTypeDaily:
-			fmt.Println(dbCredentials, auditDBCredentials)
-			return fmt.Errorf("TODO: collect analytic count daily")
+			log.Println("Start collecting daily analytic count")
+			yesterday := time.Now().UTC().AddDate(0, 0, -1)
+			err := countCollector.CollectDaily(yesterday)
+			if err != nil {
+				return err
+			}
+			log.Println("Done")
```

Comment:
Can we make this command to support the following synopsis?

```
# Collect the count for yesterday
collect-count yesterday
# Collect the count for last week
collect-count last-week
# Collect the count for last month
collect-count last-month
# Collect the daily count for the specific date
collect-count 2021-09-14
# Collect the weekly count for the specific week
collect-count 2021W40
# Collect the monthly count for the specific month
collect-count 2021-09
```

And it should be possible to run `collect-count` multiple times. The command should be smart enough to upsert the count.